freqtrade/data/converter/timeframe.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime, timezone
+from typing import Optional
+
+import ccxt
```

Comment:
From a quick spot-check - essentially all methods in this file already exist already as part of the exchange_utils file.

I don't see us duplicating such code - please remove.

Reply:
now I didn't duplicate, I extracted them from `exchange_utils` into `data/converter/timeframe.py`, please double check it

Reply:
Yes - i did double check it. Please revert, thanks. it's now in the completely wrong location.

Reply:
Also while reverting - i'd apreciate if you could actually drop the faulty / problematic commits. 

I'd not want history  of this file to be polluted by "moved out, moved back" commits (the alternative is a squash merge - but i really don't like that type of merges).

Reply:
> Also while reverting - i'd apreciate if you could actually drop the faulty / problematic commits.
I'd not want history of this file to be polluted by "moved out, moved back" commits (the alternative is a squash merge - but i really don't like that type of merges).

dropped them